Bang.Tests/Characters/PedroRamirezSpecification.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Bang.Tests.Characters
+{
+    class PedroRamirezSpecification
```

Comment:
I'm not sure that this class is necessary

Reply:
Removed